client/components/common/DropDownList.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import onClickOutside from 'react-onclickoutside';
+
+
+class DropDownList extends Component {
+  render() {
+    const { lists, classes = '', onClick } = this.props;
+
+    return (
+      <div>
+        <ul className={`${classes}__dp-wrapper`}>
```

Comment:
About the `classes` prop, I was wondering what the intention of this prop is. In its current form, it does not really allow a person re-using this component to override the styles. For example, if we pass in a custom class like `my-custom-list`, it gets concatenated with `__dp-wrapper` and then no style will be applied to the dropdown. The current setup works because `__dp-wrapper` was defined in the EventDetails page stylesheet. If we want to reuse in a different page, we would have to copy the `__dp-wrapper` styles over to the new page. This leads to code duplication. Could we make the className application more flexible?

Reply:
resolved